.github/workflows/md-check-links.yml

```
@@ -0,0 +1,28 @@
+name: Check Markdown links
+# https://github.com/gaurav-nelson/github-action-markdown-link-check
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+    types: [opened, reopened, ready_for_review, synchronize]
+    paths:
+      - "!*.md"
+      - ".github/**"
+      - "src/**"
+      - "tests/**"
+      - "requirements/**"
+      - "setup.py"
+      - "examples/**"
+      - "dockers/**"
+      - "!**/*.md"
```

```suggestion
      - ".github/workflows/md-check-links.yml"
      - "**/*.md"
```
as it MD action we shall not skip MD files :)

---

Makes sense. I have added `**/*.md` but why do you suggest `- ".github/workflows/md-check-links.yml"`?

---

I'm curious to know why `- ".github/workflows/md-check-links.yml"` is needed.

---

Because you want changes to this file to trigger itself